docs/products/postgresql/concepts/pg-connection-limits.rst

```diff
@@ -0,0 +1,28 @@
+PostgreSQL connection limits
```

Comment:
I think this could be "Connection Limits Per Plan" as a title, and be a reference item (which I've added at top level of navigation for now)

Reply:
Changed title and moved to reference